### DISCUSSION TITLE
Screen Shot

### DISCUSSION THREAD

**Discussion by cyberprophet** [ShareAnTalk/Kiwoom#23] [Show and tell]
![Screenshot_20240414-221442](https://github.com/ShareAnTalk/Kiwoom/assets/48705422/04e82777-a188-4aba-858b-035c391aa88d)
![Screenshot_20240414_221210_AnTech](https://github.com/ShareAnTalk/Kiwoom/assets/48705422/357edda8-0519-485c-95f2-249c77a8a6a9)

**Comment by cyberprophet** [comment]
![KakaoTalk_20240415_234956116](https://github.com/ShareAnTalk/Kiwoom/assets/48705422/d3072e81-4da0-49b9-9330-b7c451017a50)

**Comment by cyberprophet** [comment]
![KakaoTalk_20240416_021717670](https://github.com/ShareAnTalk/Kiwoom/assets/48705422/1d886160-9daf-47f4-8bf0-af1586e13454)

**Comment by cyberchacha** [comment]
![KakaoTalk_20240416_103836778](https://github.com/ShareAnTalk/Kiwoom/assets/134266635/c3d5b766-8c1a-4abd-b7f2-a53db810894c)

**Comment by cyberprophet** [comment]
![KakaoTalk_Photo_2024-04-17-15-30-35](https://github.com/ShareAnTalk/Kiwoom/assets/48705422/a4a6073e-67ab-4e27-be52-d346339504c5)

**Comment by cyberprophet** [comment]
![KakaoTalk_Photo_2024-04-18-02-15-18](https://github.com/ShareAnTalk/Kiwoom/assets/48705422/0473bc1e-e578-4bd6-a457-c51083e9f410)
